README.md

```diff
@@ -1,116 +1,59 @@
 
 ```
-headnode# sdc-sapi /services -X POST -d '
-{
-      "name": "picker",
-      "application_uuid": "<MANTA APPLICATION UUID>",
-      "params": {
-        "networks": [
-          "manta",
-          "admin"
-        ],
-        "ram": 256,
-        "image_uuid": "<PICKER IMAGE UUID>"
-      },
-      "metadata": {},
-      "master": true
-}'
+# manta-adm show -j -s > manta.json
 ```
 
-**Create an instance of the picker service**
+**Add the following entry to manta.json**
 
 ```
-headnode# sdc-sapi /instances -X POST -d '
-{
-      "service_uuid": "<PICKER SERVICE UUID>",
-      "params": {
-        "brand": "joyent-minimal",
-        "alias": "picker0",
-        "hostname": "picker0"
-      },
-     "metadata": {
-        "SERVICE_NAME": "picker.<DATACENTER>.<REGION>.joyent.us",
-        "MUSKIE_DEFAULT_MAX_STREAMING_SIZE_MB": 5120,
-        "MUSKIE_MAX_UTILIZATION_PCT": 90,
-        "MUSKIE_MAX_OPERATOR_UTILIZATION_PCT": 92,
-        "DATACENTER": "<DC NAME>",
-        "SDC_NAMESERVERS": [
-        {
-          "host": "<TRITON BINDER IP>",
-          "port": 2181,
-          "num": 1,
-          "last": true
-        }],
-       "SAPI_URL": "<SAPI_URL>",
-       "user-script": "#!/usr/bin/bash\n#\n# This Source Code Form is subject to the terms of the Mozilla Public\n# License, v. 2.0. If a copy of the MPL was not distributed with this\n# file, You can obtain one at http://mozilla.org/MPL/2.0/.\n#\n\n#\n# Copyright (c) 2014, Joyent, Inc.\n#\n\nset -o xtrace\nset -o errexit\nset -o pipefail\n\n#\n# To use the same convention as SDC instances, the presence of the\n# /var/svc/.ran-user-script file indicates that the instance has already been\n# setup (i.e. the instance has booted previously).\n#\n# Upon first boot, run the setup.sh script if present.  On all boots including\n# the first one, run the configure.sh script if present.\n#\nSENTINEL=/var/svc/.ran-user-script\n\nDIR=/opt/smartdc/boot\n\n\nif [[ ! -e ${SENTINEL} ]]; then\n\tif [[ -f ${DIR}/setup.sh ]]; then\n\t\t${DIR}/setup.sh\n\tfi\n\n\ttouch ${SENTINEL}\nfi\n\nif [[ -f ${DIR}/configure.sh ]]; then\n\t${DIR}/configure.sh\nfi\n"
-      }
-}'
+"picker": { "$picker_image": 1 }
 ```
 
-## Updating the Image
-
-The following steps can be used to update the image for an existing picker SAPI service
-
-**Delete the existing picker service instances**
+**Update the manta deployment**
 
 ```
-headnode# sdc-sapi /instances/<PICKER_INSTANCE_UUID> -X DELETE
-
+# manta-adm update --skip-verify-channel manta.json
 ```
 
-**Remove the old image**
+## Development
 
-```
-headnode# oldimg=$(sdc-sapi /services?name=picker | json -H [0].params.image_uuid)
-headnode# sdc-imgadm delete $oldimg
-headnode# imgadm delete $oldimg
-```
+Developer changes that are pushed to a branch of joyent/manta-picker will be automatically built by Jenkins  and the resulting image will be available in the experimental channel.  To manually build a development image, use the steps below:
```

Comment:
extra space between `Jenkins` and `and`

Reply:
That paragraph has been rewritten per Tim's comments.